tinygrad/features/search.py

```diff
@@ -11,7 +11,7 @@
 from tinygrad.tensor import Tensor
 from tinygrad.shape.symbolic import sym_infer
 
-actions = [Opt(op=OptOps.UPCAST, axis=axis, amt=amt) for amt in [0,2,3,4,5,7] for axis in range(6)]
+actions = [Opt(op=OptOps.UPCAST, axis=axis, amt=amt) for amt in [0,2,3,4,5,7,8] for axis in range(6)]
```

Comment:
how much speed is coming from this? technically we need to update db version if we change default actions

Reply:
I'll run a red and green test without it.